view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -0,0 +1,178 @@
+define(
+    [
+        'jquery',
+        'ko',
+        'mage/storage',
+        'Magento_Checkout/js/view/payment/default',
+        'Magento_Checkout/js/model/full-screen-loader',
+        'Magento_Checkout/js/action/redirect-on-success',
+        'Magento_Checkout/js/model/quote',
+        'Magento_Checkout/js/model/url-builder',
+        'Magento_Checkout/js/model/error-processor'
+    ],
+    function (
+        $,
+        ko,
+        storage,
+        Component,
+        fullScreenLoader,
+        redirectOnSuccessAction,
+        quote,
+        urlBuilder,
+        errorProcessor
+    ) {
+        'use strict';
+
+        return Component.extend({
+            defaults: {
+                template: 'Omise_Payment/payment/offsite-installment-form'
+            },
+
+            isPlaceOrderActionAllowed: ko.observable(quote.billingAddress() != null),
+
+            /**
+             * Get payment method code
+             *
+             * @return {string}
+             */
+            getCode: function () {
+                return 'omise_offsite_installment';
+            },
+
+            /**
+             * Initiate observable fields
+             *
+             * @return this
+             */
+            initObservable: function () {
+                this._super()
+                    .observe([
+                        'omiseOffsite',
+                        'installmentTerms'
+                    ]);
+
+                return this;
+            },
+
+            /**
+             * Get a checkout form data
+             *
+             * @return {Object}
+             */
+            getData: function () {
+                return {
+                    'method': this.item.method,
+                    'additional_data': {
+                        'offsite': this.omiseOffsite(),
+                        'terms': this.installmentTerms()
+                    }
+                };
+            },
+
+            /**
+             * Is method available to display
+             *
+             * @return {boolean}
+             */
+            isActive: function () {
+                return this.getOrderCurrency().toLowerCase() === 'thb' && this.getStoreCurrency().toLowerCase() === 'thb';
+            },
+
+            /**
+             * Returns Installment Terms
+             *
+             * @return {array}
+             */
+            getInstallmentTerms(id) {
+                const capabilities = checkoutConfig.payment.capabilities;
```

Comment:
Again, this should really be named 'Backends'? To match conventions in OmisePHP and Capabilities API

Reply:
what even mean backends? :) it is about getting capabilities, user capabilities, what he can do.. from logical point of view.. and naming convention it is fine IMO

Reply:
The capabilities API returns a range of things that the merchant account is capable of. The backends part of this lists the payment backends that the account is capable of using. Other parts detail the limits dictating the amounts a merchant account is capable of charging and transferring. In the API response, each capability within the capabilities returned in the `capabilities` object is contained in it's own, named, section. In the case of a lot of the code here, we are interested in just the 'backend' capabilities of the account, not all the capabilities together. For this reason, and to avoid any confusion between the various levels (API, OmisePHP, Plugin), I feel it is best we stick to using the same names as suggested/defined in the API response

Reply:
I imagine more capabilities (in their own sections) will be added into the API response in the future also

Reply:
Second that. `capabilities` is too vague.